src/systemtest/java/com/ericsson/ei/systemtest/utils/StepsUtils.java

```diff
@@ -26,18 +26,21 @@
     * @param jenkinsToken - Token to the jenkins job.
     *
     * @return boolean - If the creation was a success or not
-    *
-    * @throws JSONException
-    * @throws URISyntaxException
-    * @throws IOException
+     * @throws Exception
     */
-    public static boolean createJenkinsJob(String jenkinsJobName, String scriptFileName, String jenkinsBaseUrl, String jenkinsUsername, String jenkinsPassword, String jenkinsToken, String remremBaseUrl) throws URISyntaxException, JSONException, IOException {
+    public static boolean createJenkinsJob(String jenkinsJobName, String scriptFileName, String jenkinsBaseUrl, String jenkinsUsername, String jenkinsPassword, String jenkinsToken, String jenkinsJobXml, String remremBaseUrl) throws Exception {
         jenkinsManager = new JenkinsManager(jenkinsBaseUrl, jenkinsUsername, jenkinsPassword);
+
+        if(!jenkinsManager.pluginExists("Groovy")) {
+            jenkinsManager.installPlugin("Groovy", "2.1");
+            jenkinsManager.restartJenkins();
+        }
         String script = new String(Files.readAllBytes(Paths.get(scriptFileName)));
         script = script.replace("<REMREM_BASE_URL>", remremBaseUrl);
```

Comment:
Why all these ">" characters that need to be escaped in some situations?

Reply:
Because they are temporary and should be replaced before the script is used. It's so i can put values into the script easily.